contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,131 @@
+
+SQL insights are especially useful for when you need to do exotic or novel analysis that isn't immediately possible with existing insight types. For example, while you can use a retention insight to discover which product features keep users coming back for more, you could use an SQL insight to find which users keep coming back and identify outliers. 
+
+We can't wait to hear your feedback, and see what's possible with SQL access in PostHog. We're confident it will give engineers a unique edge, and better information, than other analytics tools.
+
+To give you a sense of what's possible with SQL, here's an example query we use to summarize data from a survey we ran to determine which countries prefer pineapple on a pizza.
+
+```
+   select properties.$geoip_country_name,
+          countIf(not properties.does_pineapple_go_on_pizza) > countIf(properties.does_pineapple_go_on_pizza)
+            ? '🍅 Does not belong on pizza'
+            : countIf(not properties.does_pineapple_go_on_pizza) = countIf(properties.does_pineapple_go_on_pizza)
+              ? '🥦 It is a tie'
+              : '🍍 Belongs on pizza'
+          as Result,
+          concat(
+             repeat('🍍', countIf(properties.does_pineapple_go_on_pizza)),
+             repeat('🍅', countIf(not properties.does_pineapple_go_on_pizza))
+          ) as Answers,
+          count() as `Number of answers`,
+          concat(
+            toString(round(countIf(properties.does_pineapple_go_on_pizza) / count() * 1000) / 10), '%'
+          ) as `Percentage pineapple`
+     from events
+    where event = 'pineapple_on_pizza_survey'
+ group by properties.$geoip_country_name
+ order by count() desc
+    limit 100
+```
+
+## The PostHog event explorer 
+We haven't introduced HogQL in isolation – we've also bought it into existing parts of PostHog, supercharging them to deliver new functionality. 
+
+The Event Explorer (previously the Live Events tab) is a great example of this. What was previously a stream of incoming events now offers wholly new ways to interact with your data and filter it into custom tables. 
+
+The event explorer enables you to view and edit the JSON source behind the table, so you can create tables in PostHog and add them to dashboards. You can also use HogQL expressions (e.g. `properties.$screen_width * $properties.screen_height`) and aggregations (e.g. `sum(properties.price)`) as columns.
+
+This is helpful not just for generating tables that summarize person and event information according to your needs, but also for exploring the data in new and totally custom ways. 
+
+You could, for example, use HogQL to run currency conversions and normalize data into a single currency in its own column – or even extrapolate and project revenue and usage figures into the future.
+
+## Custom insights
```

Comment:
```suggestion
## Custom insights

```

Reply:
Also, is this not just saving event explorer views?

Reply:
I was expecting this to be the "insight source" schema editor, rather than just saving from event explorer.

Reply:
Good question. I was just writing a twitter thread for this and chose to focus on the 'view source' feature as it seemed more interesting (to me).

@mariusandra sorry to bug you again, but your input might be useful here

Reply:
We're also slightly struggling with how to present this. The tldr is that thanks to "data exploration", everything is now represented with a JSON object that can be 1) put on a dashboard, 2) explored as an insight, etc. This includes data tables like the event explorer. You can now save those as insights, however you can also view/edit the source of traditional insights. I'm not sure what's the best way to serve this.